manifests/params.pp

```diff
@@ -23,4 +23,6 @@
   $statsd_address = '127.0.0.1:8125'
 
   $statsd_prefix = 'nsq.%s'
+
+  $msg_timeout = '15m'
```

Comment:
We don't want to set the default to 15m here so please update this to be the NSQ default which is `1m0s`. 

Then we can overwrite to `15m` as you'd like in our internal Puppet code.

This module is open source so it is not just used by Spring, but is a generic Puppet NSQ module.